README.md

```diff
@@ -1,3 +1,5 @@
+![Bitcoin Core](https://bitcoincore.org/assets/images/bitcoin_core_logo.png)
```

Comment:
```
![Bitcoin Core](doc/bitcoin_logo_doxygen.png?raw=true "Bitcoin Core")
```

Reply:
I added the small version hosted in the repo.

Thanks!